js-code/sudokuValidation.js

```diff
@@ -0,0 +1,33 @@
+arr = [
+  [0, 0, 0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 5, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0, 0, 0],
+];
+
+function validate(arr, r, c, val) {
```

Comment:
function name is too generic. Custom names should be meaningful. ```sudokuValidation``` would be fine. Any other name with good representation of what the function does would be fine too.

Reply:
```r,c``` these names are again random. Valid names would be ```row, col```.